patches/0027-Do-not-compute-tags_all-at-TF-level.patch

```diff
@@ -231,110 +231,20 @@ index 789e374885..d5a771ce81 100644
--
--	resourceTags := tftags.New(ctx, diff.Get("tags").(map[string]interface{}))
--
--	allTags := defaultTagsConfig.MergeTags(resourceTags).IgnoreConfig(ignoreTagsConfig)
--	// To ensure "tags_all" is correctly computed, we explicitly set the attribute diff
--	// when the merger of resource-level tags onto provider-level tags results in n > 0 tags,
--	// otherwise we mark the attribute as "Computed" only when there is a known diff (excluding an empty map)
--	// or a change for "tags_all".
--	// Reference: https://github.com/hashicorp/terraform-provider-aws/issues/18366
--	// Reference: https://github.com/hashicorp/terraform-provider-aws/issues/19005
--
--	if !diff.GetRawPlan().GetAttr("tags").IsWhollyKnown() {
--		if err := diff.SetNewComputed("tags_all"); err != nil {
--			return fmt.Errorf("setting tags_all to computed: %w", err)
--		}
--		return nil
--	}
--
--	if diff.HasChange("tags") {
--		_, n := diff.GetChange("tags")
--		newTags := tftags.New(ctx, n.(map[string]interface{}))
--
--		if newTags.HasZeroValue() {
--			if err := diff.SetNewComputed("tags_all"); err != nil {
--				return fmt.Errorf("setting tags_all to computed: %w", err)
--			}
--		}
--
--		if len(allTags) > 0 && (!newTags.HasZeroValue() || !allTags.HasZeroValue()) {
--			if err := diff.SetNew("tags_all", allTags.Map()); err != nil {
--				return fmt.Errorf("setting new tags_all diff: %w", err)
--			}
--		}
--
--		if len(allTags) == 0 {
--			if err := diff.SetNew("tags_all", allTags.Map()); err != nil {
--				return fmt.Errorf("setting new tags_all diff: %w", err)
--			}
--		}
--	} else if !diff.HasChange("tags") {
--		if len(allTags) > 0 && !allTags.HasZeroValue() {
--			if err := diff.SetNew("tags_all", allTags.Map()); err != nil {
--				return fmt.Errorf("setting new tags_all diff: %w", err)
--			}
--			return nil
--		}
--
--		var ta tftags.KeyValueTags
--		if tagsAll, ok := diff.Get("tags_all").(map[string]interface{}); ok {
--			ta = tftags.New(ctx, tagsAll)
--		}
--		if len(allTags) > 0 && !ta.DeepEqual(allTags) && allTags.HasZeroValue() {
--			if err := diff.SetNewComputed("tags_all"); err != nil {
--				return fmt.Errorf("setting tags_all to computed: %w", err)
--			}
--			return nil
--		}
--	} else if tagsAll, ok := diff.Get("tags_all").(map[string]interface{}); ok {
--		ta := tftags.New(ctx, tagsAll)
--		if !ta.DeepEqual(allTags) {
--			if allTags.HasZeroValue() {
--				if err := diff.SetNewComputed("tags_all"); err != nil {
--					return fmt.Errorf("setting tags_all to computed: %w", err)
--				}
--			}
--		}
--	} else if len(diff.Get("tags_all").(map[string]interface{})) > 0 {
--		if err := diff.SetNewComputed("tags_all"); err != nil {
--			return fmt.Errorf("setting tags_all to computed: %w", err)
--		}
--	} else if diff.HasChange("tags_all") {
--		if err := diff.SetNewComputed("tags_all"); err != nil {
--			return fmt.Errorf("setting tags_all to computed: %w", err)
--		}
--	}
--
- 	return nil
- }
++	// Pulumi handles tags differently for Plugin Framework.
```

Comment:
@corymhall yes so as part of this PR I endeavored to minimize subsequent patch conflicts for us. What we want to do is to make this function a no-op. The function keeps changing. If we do this as we did before by deleting the code, the changes generate more conflicts. With this new method we just short-circuit it to nil right upfront, but let the rest of the code drift upstream. I think this is better.